Back End/MemorizeWords/MemorizeWords/Infrastructure/Persistance/Repository/WordRepository.cs

```diff
@@ -129,16 +129,33 @@ private static void ValidationAddUpdateWord(WordAddRequest wordAddRequest)
 
         public async Task<List<QuestionWordResponse>> GetQuestionWordsAsync()
         {
-            var randomWords = await Queryable().Where(x => !x.IsLearned)
-                                   .OrderBy(x => Guid.NewGuid())
-                                   .Take(20)
-                                   .Select(x => new QuestionWordResponse()
-                                   {
-                                       Word = x.Word,
-                                       Id = x.Id,
-                                       WritingInLanguage = x.WritingInLanguage,
-                                       Meaning = x.Meaning
-                                   }).ToListAsync();
+            var learningLanguageWordCount = new Random().Next(0, 21);
```

Comment:
new Random().Next(0, 21)

make it method and give a meaningful name and use 20 value from constant value